packages/barcode-scanning/src/definitions.ts

```diff
@@ -13,6 +13,18 @@ export interface BarcodeScannerPlugin {
    * @since 0.0.1
    */
   stopScan(): Promise<void>;
+  /**
+   * Pause scanning for barcodes.
+   *
+   * @since 7.3.1
```

Comment:
```suggestion
   * @since 7.5.0
```

If we manage to release it this week, it would be version 7.5.0.